frontend/src/pages/LogsPage/LogsTable/LogsTable.tsx

```diff
@@ -119,56 +119,48 @@ const LogsTableInner = ({
 	const queryTerms = parseLogsQuery(query)
 	const [expanded, setExpanded] = useState<ExpandedState>({})
 
-	const columns = React.useMemo<ColumnDef<LogEdge>[]>(
```

Comment:
Didn't see a reason to keep this memoized. It seems to only make the typing more difficult to manage. I did a clicktest on the reflame preview and don't see any issues.

Reply:
I added the `useMemo` originally here: https://github.com/highlight/highlight/commit/21771a06efa1279c79af2879bf4847c9fed2b8d8#diff-ee6f428905a95e722c24e36e545d285643f49fc7d84333333089c32ec2bbfaeeR44

But I believe it was just copy/pasted from some example.

I also tracked down the eslint ignore rule to here: https://github.com/highlight/highlight/commit/cedbdd740620cdcb3e6fb79e5bbf792dbe156bce#diff-ee6f428905a95e722c24e36e545d285643f49fc7d84333333089c32ec2bbfaeeR126-R127

since we needed the columns to rerender when the query was updated.

Since it's annoying to add eslint rules like this, it's best to avoid memoization until we can identify a reason to add it.